src/main/fc/config.h

```diff
@@ -33,7 +33,7 @@
 
 typedef enum {
     FEATURE_RX_PPM = 1 << 0,
-    //FEATURE_VBAT = 1 << 1,
+    //UNUSED = 1 << 1,
```

Comment:
I will remove them, I did let the commented lines in to quickly identify which spots are currently not used and can be used for future features.